index.html

```diff
@@ -107,16 +107,26 @@ <h1>EmulatorJS Demo</h1>
 
         <script>
             var enableDebug = false;
+            var enableThreads = false;
             const queryString = window.location.search;
             const urlParams = new URLSearchParams(queryString);
             if (urlParams.get('debug') == 1)
                 enableDebug = true;
+
+            if (urlParams.get('threads') == 1)
+                enableThreads = true;
             
             if (enableDebug == true) {
                 console.log("Debug is enabled");
             } else {
                 console.log("Debug is disabled");
             }
+
+            if (enableThreads == true) {
```

Comment:
No need for the ` == true`

Reply:
Simplified the code by not doing the same test twice.